utils/seed_asset.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+
+import django
+
+project_dir = os.path.dirname(
+    os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(project_dir)
+
+from utils.helpers import is_valid_file  # noqa
+from utils.asset.post_scripts import (
+    post_asset_make, post_asset_category,
+    post_asset_model_no, post_asset, post_asset_types,
+    post_asset_subcategory
+)  # noqa
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+django.setup()
+
+
+if __name__ == '__main__':
+    filename = input('Enter csv filename (without .csv): ').strip()
+    filepath = os.path.abspath(os.path.join(os.path.dirname(__file__))) + '/'  # noqa
```

Comment:
The input file ought to be loaded from the root of the project. Isn't that the case @egichuri ?

Reply:
i think we can have it in the utils folder since we'll probably only use it once

Reply:
Sounds good!